cluster_experiments/washover.py

```diff
@@ -10,6 +10,27 @@
 class Washover(ABC):
     """Abstract class to model washovers in the switchback splitter."""
 
+    def _validate_columns(
+        self,
+        df: pd.DataFrame,
+        truncated_time_col: str,
+        cluster_cols: List[str],
+        original_time_col: str,
+    ):
+        if original_time_col not in df.columns:
+            raise ValueError(
```

Comment:
I'd add tests that validate that these conditions are indeed raising an error with pytest.raises. let me know if you need help with that

Reply:
I'd also add a docstring to this method